Chapter08/recipe-06/cxx-example/src/evolution/evolution.cpp

```diff
@@ -0,0 +1,29 @@
+#include <string>
+#include <vector>
+
+#include "evolution.hpp"
```

Comment:
Ditto. This should be the first `include`-s Suggested read: http://www.cplusplus.com/forum/articles/10627/